python/pip_install/pip_compile.py

```diff
@@ -57,6 +57,7 @@
 update_command = os.getenv("CUSTOM_COMPILE_COMMAND") or "bazel run //%s:%s" % (update_target_pkg, update_target_name)
 
 os.environ["CUSTOM_COMPILE_COMMAND"] = update_command
+os.environ["PIP_CONFIG_FILE"] = os.getenv("PIP_CONFIG_FILE") or "/dev/null"
```

Comment:
not sure what windows support is like so `/dev/null` might need to change?

Reply:
Hello, Windows equivalent is 'nul'. Python has `os.devnull` abstracting that https://docs.python.org/3/library/os.html#os.devnull

Reply:
thanks!